server/tool/src/index.ts

```diff
@@ -258,8 +258,19 @@ export async function upgradeModel (
       logger.log('model transactions inserted.', { workspaceId: workspaceId.name, count: insert.insertedCount })
       await progress(20)
     }
+    const newModel = [
+      ...txes,
+      ...Array.from(
+        prevModel.model.filter(
+          (it) =>
+            it.modifiedBy !== core.account.System &&
```

Comment:
`===` ?

Reply:
Why? We need all user transactions, all system are deleted.

Reply:
Or is required.

Reply:
Yep, it was just not in sync with https://github.com/hcengineering/platform/blob/main/server/tool/src/index.ts#L239. With `or` it seems fine now.